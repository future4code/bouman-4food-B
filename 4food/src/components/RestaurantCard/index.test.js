import React from 'react';
import renderer from 'react-test-renderer';
import { RestaurantCard } from './index'

it('Snapshot RestaurantCard', ()=> {
    const tree = renderer
    .create(<RestaurantCard></RestaurantCard>)
    .toJSON();
    expect(tree).toMatchSnapshot()
})