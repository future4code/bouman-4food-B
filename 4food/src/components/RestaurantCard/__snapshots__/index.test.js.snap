// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot RestaurantCard 1`] = `
<div
  className="sc-bdVaJa iKMTul"
>
  <img
    className="sc-ifAKCX bHeiqT"
    src="image.png"
  />
  <div
    className="sc-bwzfXH kWmsFi"
  >
    <div
      className="sc-htpNat klhMYs"
    >
      <p
        className="sc-EHOje dqOFZr"
      >
        Vinil Butant√£
      </p>
    </div>
    <div
      className="sc-bxivhb hgknfb"
    >
      <p
        className="sc-bZQynM hOpKhM"
      >
        50 - 60 min
      </p>
      <p
        className="sc-bZQynM hOpKhM"
      >
        Frete R$6,00
      </p>
    </div>
  </div>
</div>
`;
